src/Chat/ChatContainer.js

```diff
@@ -17,7 +17,7 @@ export default (props) => {
   const list = props.chatData.map((item, index) => {
     return (
       <ChatItemContainer
```

Comment:
list보다는 좀 더 구체적인 이름이면 좋을 것 같아요. `ChatItemList` 도 괜찮아보이네용.
(대문자인 이유는 JSX 컴포넌트들의 리스트라는 걸 암시)
```suggestion
  const ChatItemList = props.chatData.map((chatItem, index) => {
    return (
      <ChatItemContainer
```